seahorn/sea_base.yaml

```diff
@@ -1,45 +0,0 @@
-verify_options:
```

Comment:
do not expect this PR to remove this file

Reply:
This file is a no-op since `sea.yaml` does not link to it.

Reply:
you mean that this file is not used? If so, it should be removed in a separate PR to make review easier

Reply:
Fixed.